src/NzbDrone.Core/Notifications/Trakt/Trakt.cs

```diff
@@ -26,45 +30,24 @@ public Trakt(ITraktProxy proxy, INotificationRepository notificationRepository,
 
         public override void OnDownload(DownloadMessage message)
         {
-            var payload = new TraktAddMoviePayload
+            var payload = new TraktCollectMoviesResource
             {
-                Movies = new List<TraktAddMovie>()
+                Movies = new List<TraktCollectMovie>()
             };
 
-            var width = message.MovieFile.MediaInfo?.Width ?? 0;
-            var mediaInfo = "";
+            var traktResolution = MapResolution(message.MovieFile.Quality.Quality.Resolution);
+            var mediaType = MapMediaType(message.MovieFile.Quality.Quality.Source);
+            var audio = MapAudio(message.MovieFile);
 
-            if (width >= 3200)
-            {
-                mediaInfo = "uhd_4k";
-            }
-
-            if (width >= 1800)
-            {
-                mediaInfo = "hd_1080p";
-            }
-
-            if (width >= 1200)
-            {
-                mediaInfo = "hd_720p";
-            }
-
-            if (width >= 700)
-            {
-                mediaInfo = "sd_576p";
-            }
-
-            if (width > 0)
-            {
-                mediaInfo = "sd_480p";
-            }
-
-            payload.Movies.Add(new TraktAddMovie
+            payload.Movies.Add(new TraktCollectMovie
             {
                 Title = message.Movie.Title,
                 Year = message.Movie.Year,
                 CollectedAt = DateTime.Now,
-                Resolution = mediaInfo,
+                Resolution = traktResolution,
+                MediaType = mediaType,
+                AudioChannels = MediaInfoFormatter.FormatAudioChannels(message.MovieFile.MediaInfo).ToString(),
```

Comment:
AudioChannels should often map 1:1 with what Trakt is expecting, but I found the below possible exceptions. I have redirected the mappings to channel counts I think are appropriate

    #Below is when DTS-X is used and MediaInfo does not give object counts. These are definitely 7.1
    if audio_channel_count == 8.0:
        audio_channels = "7.1"
    #Below incorrect count can sometimes be 6.1 for DTS-HR tracks, but the vast majority of time, it is for 7.1 tracks
    #It happens when channels_original and channels differ, and all my cases were DTS-HD. Out of 17 such cases, only 
    #3 were actually 6.1, rest were 7.1. Two of the 6.1 were DTS-ES, the other was PCM
    elif audio_channel_count == 6.0 and audio == "dts_ma":
        audio_channels = "7.1"
    elif audio_channel_count == 6.0 and audio != "dts_ma":
        audio_channels = "6.1"
    #Not sure why this happens, but I noticed a few older 1.0 PCM tracks coming as 0.0 channel count in Radarr
    elif audio_channel_count == 0.0:
        audio_channels = "1.0"